cloudnew/ec2_client.go

```diff
@@ -0,0 +1,193 @@
+package cloudnew
```

Comment:
its probably safe to put this in the existing cloud package with different type names (and some changes to the factory) that we can twiddle when we delete the old ones.)